syntax = "proto3";

package client;

message ClHandshake {
  int32 id = 1;
}